mlguess/keras/losses.py

```diff
@@ -124,13 +111,19 @@ def get_config(self):
 
 
 def GaussianNLL(y, y_pred, reduce=True):
```

Comment:
we should change the name of this function to be lowercase and separated by underscores: `gaussian_nll`